src/modules/workspaces/workspaces.controller.ts

```diff
@@ -348,4 +348,51 @@ export class WorkspacesController {
       changeUserRoleBody,
     );
   }
+
+  @Get(':workspaceId/members/:memberId')
+  @ApiBearerAuth()
+  @ApiOperation({
+    summary: 'Gets workspace member details',
+  })
+  @ApiParam({ name: 'workspaceId', type: String, required: true })
+  @ApiParam({ name: 'memberId', type: String, required: true })
+  @ApiOkResponse({
+    description: 'Details of workpace member',
```

Comment:
A typo here 'Details of the workspace members' instead of 'Details of workpace member'